File: problemas/1100/1164.cs
using System;

namespace uri1164
{
    internal static class Program
    {
        private static void Main()
        {
            int n = int.Parse(Console.ReadLine());

            for (int i = 0; i < n; i++)
            {
                int x = int.Parse(Console.ReadLine());
                int soma = 0;

                for (int j = 1; j < x; j++)
                {
                    if (x % j == 0)
                    {
                        soma += j;
                    }
                }

                Console.WriteLine(x + (soma == x ? " eh perfeito" : " nao eh perfeito"));
            }
        }
    }
}

File: problemas/1100/1164.php
<?php

$n = readline();

for ($i = 0; $i < $n; $i++) {
    $x = readline();
    $soma = 0;

    for ($j = 1; $j < $x; $j++) {
        if ($x % $j == 0) {
            $soma += $j;
        }
    }

    echo $x . ($soma == $x ? " eh perfeito\n" : " nao eh perfeito\n");
}